financialaid/admin.py

```diff
@@ -31,11 +31,13 @@ def has_delete_permission(self, *args, **kwargs):  # pylint: disable=unused-argu
 class TierProgramAdmin(admin.ModelAdmin):
     """Admin for TierProgram"""
     model = TierProgram
+    list_display = ('__str__', 'discount_amount', 'income_threshold', 'current')
```

Comment:
I would display the `tier__name` and `program__title` explicitly and not rely on the formatting of `__str__`